usaddress/__init__.py

```diff
@@ -61,72 +61,81 @@
                   'northeast', 'northwest', 'southeast', 'southwest'])
 
 STREET_NAMES = {
-    'allee', 'alley', 'ally', 'aly', 'anex', 'annex', 'annx', 'anx', 'arc',
```

Comment:
The diff is not super helpful here, so I pulled out what seem to be your new contributions:

```python
{'bl', 'causway', 'ci', 'ck', 'cr', 'crecent', 'cresent', 'crscnt',
'crssing', 'crt', 'ex', 'havn', 'height', 'hgts', 'la', 'lanes',
'lp', 'mi', 'mile', 'mn', 'overlook', 'ovlk', 'pk', 'plaines', 'prarie',
'pw', 'pwy', 'ri', 'rise', 'rn', 'strave', 'te', 'tl', 'tpk', 'tr', 'trpk',
'turn', 'xc', 'xg'}
```

Is that correct?